app/views/layouts/exercise_inputs/editors/_multiple_files.html.erb

```diff
@@ -0,0 +1,37 @@
+<div class="field form-group editor-code">
+
+  <div class="editor-files">
+    <span class="files-tabs">
+      <ul class="nav nav-tabs">
+        <% @files.each_with_index do |filename, index| %>
+            <li role="presentation" class="<%= 'active' if index == 0 %>" data-target="#editor-file-<%= index %>" tabindex='0' data-toggle='tab'>
+              <a href="#"><%= filename %></a>
+            </li>
+        <% end %>
+      </ul>
+    </span>
+    <div class="tab-content">
+      <% @files.each_with_index do |filename, index| %>
+          <div role="tabpanel" class="tab-pane mu-input-panel <%= 'fade in active' if index == 0 %>" id="editor-file-<%= index %>">
+            <%= form.editor "content[#{filename}]", 'dynamic',
+                            placeholder: t(:editor_placeholder),
+                            class: 'form-control editor',
+                            value: @file_content[filename],
+                            data: {lines: 17} %>
+          </div>
+      <% end %>
+    </div>
+
+    <a class="editor-resize multiple-files" title="<%= t(:fullscreen) %> (F11)">
+      <span class="fa-stack fa-lg">
+        <i class="fa fa-square-o fa-stack-2x"></i>
+        <i class="fa fa-expand fa-stack-1x"></i>
```

Comment:
Does this work with a console, too? And with a `extra_code`?

Reply:
Yes